app-cdr/burn-cd/burn-cd-1.8.1-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-single-r1
+
+DESCRIPTION="Smart console frontend for the cdrkit/cdrtools & dvd+rw-tools"
+HOMEPAGE="https://github.com/aglyzov/burn-cd/"
+SRC_URI="https://github.com/aglyzov/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
```

Comment:
You can't commit straight to stable. Use `ekeyword ~all` on this ebuild before committing.